pkg/controller/accountclaim/reuse.go

```diff
@@ -141,7 +140,7 @@ func (r *ReconcileAccountClaim) resetAccountSpecStatus(reqLogger logr.Logger, re
 	reusedAccount.Status.Claimed = false
 	reusedAccount.Status.Reused = true
 	conditionMsg := fmt.Sprintf("Account Reuse - %s", conditionStatus)
-	account.SetAccountStatus(reqLogger, reusedAccount, conditionMsg, accountState, conditionStatus)
+	utils.SetAccountStatus(reqLogger, reusedAccount, conditionMsg, accountState, conditionStatus)
```

Comment:
Similar comment here: it's worrisome that accountclaim is updating Account CRs. But in this case, there would be nothing to trigger the account controller to do this status update.

Again, this is latent, but something that should probably be looked at for the future. It's possible to make the Account controller watch for AccountClaim changes as well (would have to use a map func, since the ownership is going the wrong way). Not sure if that helps here - i.e. if there's an AccountClaim change that the Account controller could recognize in order to set this status.

Reply:
:+1: